eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
@benjaoming also, isn't this the other way round, that you predefine those in your app's settings(?); I don't think it makes sense to predefine it in project settings, we should be overriding it here.